src/ast.rs

```diff
@@ -462,8 +463,9 @@ pub fn net_to_runtime(rt_net: &mut run::Net, net: &Net) {
   }
 }
 
-pub fn book_to_runtime(book: &Book) -> run::Book {
+pub fn book_to_runtime(book: &Book, call_native: run::CallNative) -> run::Book {
```

Comment:
This adds a call_native function to be easier to add a JIT generated function here